### PR TITLE
Add tvOS support

### PR DESCRIPTION
**Pull request by onmyway133**

### CODE REVIEW

File: Imaginary_tvOSTests/Imaginary_tvOSTests.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//  Imaginary_tvOSTests
+//
+//  Created by Khoa Pham on 22.09.2017.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
+//
+
+import XCTest
+
+class Imaginary_tvOSTests: XCTestCase {
+
```
Comment (vadymmarkov) [inline Imaginary_tvOSTests/Imaginary_tvOSTests.swift]: If we don't have tests I would remove this.

File: Imaginary-tvOSTests/Imaginary_tvOSTests.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+//
+//  Created by Khoa Pham on 22.09.2017.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
+//
+
+import XCTest
+@testable import Imaginary_tvOS
+
+class Imaginary_tvOSTests: XCTestCase {
+
```
Comment (vadymmarkov) [inline Imaginary-tvOSTests/Imaginary_tvOSTests.swift]: What's the difference between `Imaginary-tvOSTests` and `Imaginary_tvOSTests` folders?

File: Imaginary-tvOS/Imaginary_tvOS.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+//
```
Comment (vadymmarkov) [inline Imaginary-tvOS/Imaginary_tvOS.h]: I think it can be removed.

**Review by vadymmarkov — APPROVED** [review]

File: Imaginary_tvOSTests/Imaginary_tvOSTests.swift (hunk 1 above)
Comment (onmyway133) [reply Imaginary_tvOSTests/Imaginary_tvOSTests.swift]: 👍

File: Imaginary-tvOSTests/Imaginary_tvOSTests.swift (hunk 2 above)
Comment (onmyway133) [reply Imaginary-tvOSTests/Imaginary_tvOSTests.swift]: I have no idea. I will clean this up

**Pull request merged**